### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by varela-2021**

### CODE REVIEW

File: A5_2practice_2.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+	public static void main(String[] args) {
+		
+		boolean isPrime;
+		boolean isPerfect;
+	
+		Scanner scan= new Scanner(System.in);
+		System.out.println("Enter any number:");
+		//capture the input in an integer
+		int n=scan.nextInt();
```
Comment (samuel-soubeyran) [inline A5_2practice_2.java:12]: you need a loop that "repeatedly display a menu allowing the user to choose whether to display the prime numbers 1-1000, display the perfect numbers 1-10,000, or quit"

File: A5_2practice_2.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+			System.out.println(n  +" is a perfect number");
+	        }
+		
+	     
+	    if (isPrime = true){ 
+	            System.out.println(n +" is a prime number");
+        }
+	}
+		
+		public static boolean isPrime(int n, boolean isPrime) {
```
Comment (samuel-soubeyran) [inline A5_2practice_2.java:24]: isPrime should just take an int and return a boolean.

File: A5_2practice_2.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+			
+			if (n %2!=0) {
```
Comment (samuel-soubeyran) [inline A5_2practice_2.java:26]: this check if the number is odd, not if it is prime.

File: A5_2practice_2.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+		
+		boolean isPrime;
+		boolean isPerfect;
+	
+		Scanner scan= new Scanner(System.in);
+		System.out.println("Enter any number:");
+		//capture the input in an integer
+		int n=scan.nextInt();
+	       
+	if ( isPerfect= true){
```
Comment (samuel-soubeyran) [inline A5_2practice_2.java:14]: this doesn't make sense. You should remove

File: A5_2practice_2.java [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+			System.out.println(n  +" is a perfect number");
+	        }
+		
+	     
+	    if (isPrime = true){
```
Comment (samuel-soubeyran) [inline A5_2practice_2.java:19]: same doesn't make sense. You should remove.

File: A5_2practice_2.java [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+		public static boolean isPrime(int n, boolean isPrime) {
+			
+			if (n %2!=0) {   
+			return true;
+		}
+			return isPrime;
+		}
+
+
+	public static boolean isPerfect(int n, boolean isPerfect)
```
Comment (samuel-soubeyran) [inline A5_2practice_2.java:33]: this is good. Can you just cleanup the indentation and brackets so that it's easier to read

Comment (samuel-soubeyran) [reply A5_2practice_2.java:33]: also you should just pass in an int, not the boolean isPerfect.

File: A5_2practice_2.java (hunk 1 above)
Comment (samuel-soubeyran) [reply A5_2practice_2.java:12]: you can do a prompt such as:
"type 1 to display if the number is prime.
type 2 to display if the number is perfect
type 0 to quit"
then if the user enter 1 or 2 then prompt:
"type your number"
and call the right method (isPrime or isPerfect).

File: A5_2practice_2.java (hunk 6 above)
Comment (varela-2021) [reply A5_2practice_2.java:33]: Thank you Samuel! 
I'll work on it.

**Pull request closed without merging**